staff_features/resources/resource_container_instance.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Resource Container Instance
+  Background:
+    Given an administrator user is logged in
+      And a Resource has been created
+      And the Resource is opened in edit mode
+  Scenario: Add container instance
+     When the user clicks on 'Instances'
+      And the user clicks on 'Add Container Instance'
+      And the user selects 'Accession' from 'Type' in the 'Instances' form
+      And the user clicks on the first dropdown in the "Instances" form
```

Comment:
I think this step is not required

Reply:
The `first dropdown` refers to the dropdown arrow icon in the form (looks like an inverted `caret`). The `dropdown menu` below, is the actual menu that appears after clicking on the arrow, which the user clicks on either `Create` or `Browse`. The same step has already been used and discussed in a previous PR. Currently we are going to use the word `first`, however in the future we are going to find a more solid way of describing the dropdown icons with no text, since, a sub-record or linked record form may have more than one dropdown arrows with no label, and we need a concrete way to refer to them.